src/stores/businessStore.ts

```diff
@@ -83,9 +83,15 @@ export const useBusinessStore = defineStore('business', {
       return state.businessInfo.lastDirectorChangeDate
     },
 
-    /** The legal name. */
+    /** The legal name, or operating name if is firm. */
     getLegalName (state: BusinessStateIF): string {
-      return state.businessInfo.legalName
+      return this.isFirm && this.getOperatingName ? this.getOperatingName : state.businessInfo.legalName
```

Comment:
Please put `this.isFirm && this.getOperatingName` in parentheses. Ternaries are nice except when they look more complicated than if-then-else 😉